populate_database.py

```diff
@@ -26,25 +32,11 @@
 if Event.query.first() is None:
     print('creating event')
     event = Event(
-        name='Workshop 2019',
+        name='Toolbox Workshop 2019',
```

Comment:
should be 2020

Reply:
This is just a script for local testing,  nothing to do with the actual deployed website